libpod/boltdb_state_internal.go

```diff
@@ -304,6 +304,16 @@ func (s *BoltState) getContainerFromDB(id []byte, ctr *Container, ctrsBkt *bolt.
 	}
 	ctr.lock = lock
 
+	if ctr.config.OCIRuntime == "" {
+		ctr.ociRuntime = s.runtime.defaultOCIRuntime
+	} else {
+		ociRuntime, ok := s.runtime.ociRuntimes[ctr.config.OCIRuntime]
+		if !ok {
```

Comment:
we support specifying the runtime by an absolute path, we needed it to avoid a breaking change.  When the runtime starts with `/` we use directly the specified path without any lookup.  Should we take care of this case here and skip the error when `ctr.config.OCIRuntime[0] == '/'` ?

Reply:
Should be fixed

Reply:
Actually, didn't quite catch the case where older containers might be stored with paths starting with a `/`... For that, I think we can try grabbing the basename and looking up the runtime by that.

Reply:
Alright, really fixed now

Reply:
that is a good fix, thanks.